src/main/resources/avro/common.avdl

```diff
@@ -2,6 +2,325 @@
 /**
 This protocol defines common types used in the other GA4GH protocols. It does
 not have any methods; it is merely a library of types.
+
```

Comment:
@fnothaft Are you sure? According to [CONTRIBUTING.md](https://github.com/ga4gh/schemas/blob/master/CONTRIBUTING.md) it looks like the comment is supposed to be at the same level as the delimeters, which are at the same level as the surrounding code, which is what I've done for the top comment in common.avdl.

Or are you talking about something else?